src/link/simulationlink.h

```diff
@@ -23,7 +23,7 @@ class SimulationLink : public AbstractLink
      * @return true
      * @return false
      */
-    bool isWritable() final { return false; };
+    bool isWritable() final { return false; }
```

Comment:
Avoid style changes

Reply:
This is your preference (no semi colon). You want a new commit?

Reply:
I'll put the style changes in different commits.

Reply:
I'm dropping them for expediency.